Ntuplizer/plugins/GenParticlesNtuplizer.cc

```diff
@@ -18,86 +19,166 @@ GenParticlesNtuplizer::~GenParticlesNtuplizer( void )
 
 //===================================================================================================================        
 void GenParticlesNtuplizer::fillBranches( edm::Event const & event, const edm::EventSetup& iSetup ){
-  //chunk to remove those events with no jspi if that analysis is chosen
-  std::vector<int> doJpsi_;
-  if(isJpsiEle_) {
-    doJpsi_ = nBranches_->IsJpsiEle;
-  }else if(isJpsiMu_){
-    doJpsi_ = nBranches_->IsJpsiMu;
-  }
+  
 
-  if(size(doJpsi_)==0) return;
```

Comment:
@cgalloni sorry I missed this before, but why did you remove this line? I thought it would be helpful, not to save all Gen info ... otherwise the file becomes too big ...